### ISSUE TITLE
Choosing a license: Permissive vs Restrictive

### ISSUE THREAD

**Issue by coolharsh55** [kasnder/app-consent-android#10]
tldr; implications of license used (GPL v3) may restrict adoption and use in typical use-cases; a more permissive license such as GPL v2 or Apache 2.0 may be better

Hi. Use of GPL v3 as the License will require published Apps to be published with their source made available (AFAIK), which is typically not the norm since most apps are not open-sourced. Since the aim of this work is to encourage better handling of (personal) data in the interests of data protection and privacy, a license that facilitates integration more easily would result in more adoption and use. 

Ideally, if there is a need to nudge sharing back improvements, the GPL v2 may be a more palatable option; or if the intention is to have this work be used as widely as possible, the Apache 2.0 license may be suitable (and is the more preferred option for such work since its easier to figure out implications of reuse).

**Comment by kasnder** [comment]
Good question about license! The reason why I picked a restrictive license is for now that the project is still pretty early in development, and I'm not quite sure yet where it's headed. But if this project should become more suitable for production, I completely agree that another license might be more appropriate!
